apps/test/unit/templates/studioHomepages/TeacherHomepageTest.js

```diff
@@ -144,9 +144,9 @@ describe('TeacherHomepage', () => {
     assert(!wrapper.find('Notification').exists());
   });
 
-  it('renders a CensusTeacherBanner if showCensusBanner is true', () => {
+  it('does not render a CensusTeacherBanner even if showCensusBanner is true', () => {
     const wrapper = setUp({showCensusBanner: true});
-    assert(wrapper.find('CensusTeacherBanner').exists());
+    assert.equal(wrapper.find('CensusTeacherBanner').length, 0);
```

Comment:
Could you add a comment similar to the one you added to TeacherHomepage.jsx explaining why this is please? I imagine if someone was looking through this it could be confusing why the varibale isn't doing what it is labeled to do.

Reply:
Yes definitely!